File: workers/run_fixtures.py
"""Run the local SideQuest fixture worker."""

from __future__ import annotations

from lib.store import Store
from workers.fixtures import load_fixtures


def main() -> None:
    counts = load_fixtures(Store.create())
    print(
        "fixture cycle: "
        f"landmarks={counts['landmarks']} events={counts['events']} captions={counts['captions']}"
    )


if __name__ == "__main__":
    main()

File: scripts/run-fixtures.ts
/** Run the local SideQuest fixture worker. */

import { createStore } from '../lib/server/store.ts';
import { loadFixtures } from './fixtures.ts';

const counts = await loadFixtures(createStore());

console.log(
  'fixture cycle: ' +
    `landmarks=${counts.landmarks} events=${counts.events} captions=${counts.captions}`,
);
